test/literate_tabbed.litcoffee

```diff
@@ -69,13 +70,15 @@ if true
 
 	test "should ignore code blocks inside HTML", ->
 		eq executed, false
+		testsCount++
 
 ---
 
-*   A list item with a code block:
+*   A list item followed by a code block:
 
-		test "basic literate CoffeeScript parsing", ->
-			ok yes
+	test "basic literate CoffeeScript parsing", ->
```

Comment:
It’s not nested. Imagine there was a `---` or another regular unindented paragraph between the list item line and the code block.